static/src/stylesheets/_vars.scss

```diff
@@ -30,6 +30,8 @@ $right-column:     gs-span(4);
 $left-column:      gs-span(2); // Grows to 3 columns on wide viewports
 $left-column-wide: gs-span(3);
 
+$veggie-burger-small: 42px;
+$veggie-burger-medium: 52px;
```

Comment:
This is used in two separate files `footer.scss and header.scss`. There is very similar styling for the back to top button in the footer, and the veggie burger in the header